compiler/one-cmds/how-to-use-one-commands.txt

```diff
@@ -20,8 +20,73 @@ Compilation flow for NPU
 4) one-codegen will compile to binary codes.
 
 
+common features
+---------------
+
+[configuration file]
+
+You can run one-commands with configuration file as well as command line parameters. The con-
+figuration file should be written with the options the one-commands need to run.
+
+```
+# configuration_file.cfg
+
+[The_driver_you_want_to_run]
+input_path=/input/path/to/convert
+output_path=...
+option_0=...
+option_1=...
+...
+
+```
+
+You can see a template file for how to write a configuration file in `one-build.template.cfg`.
+
+[options to write]
+
+Sometimes you want to change certain options without touching the configuration file. If you
+pass the option directly to the command line, the option is processed prior to the configura-
+tion file. A list of options can be found in each driver's help message with `-h` option.
+
+e.g. $ ./one-import tf -C my-conf.cfg -i path/to/overwrite.pb
+
+
+one-build
+---------
+
+one-build is an integrated driver that can execute one-commands at once. It's nice to run each
+driver individually, but sometimes you'll want to put together the most frequently used comman-
+ds and run them all at once. You can do this with one-build and its configuration file.
```

Comment:
We don't have to jalign justify. Please split before `commands`

Reply:
@seanshpark Okay. I just wanted to make it align. Being neat freak:)

Reply:
Other blocks didn't do it like this.